### ISSUE TITLE
Disposals Is Broken On Meta

### ISSUE THREAD

**Issue by GoldenAlpharex** [Skyrat-SS13/Skyrat-tg#2867]
Reporting client version: 513.1539

## Round ID:
583

## Testmerges:
- [[NON-MODULAR] [TM Only] Added the SQL logging.. waf](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2668)
- [DS-1 Assault Group + Skyrat Modular Map System [READY FOR TESTING, MODULAR]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2317)
- [[WIP] Combat rework - less lethality, less hardstuns, nonlethal takedowns. Focus on combat ineffectivity](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1285)
- [[TESTMERGE] Makes icon2html not work](https://github.com/Skyrat-SS13/Skyrat-tg/pull/948)
- [[TM] Disables sentient disease - pending rebalancing/tweaks](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2030)
- [[Semi-Modular] Oldstation (Charlie station) Additions](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2706)
- [[DNM] Cryo on core maps [TESTMERGE ONLY]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1783)
- [[Ready] Maintenance Loot Overhaul](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2470)
- [[TM ONLY] Tick test to reduce lag](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2443)

## Reproduction:
Basically, there's missing pipes around Surgery C in Meta: Outside of the room itself, in maint (it's basically to the right of the monkey pen in Virology), and then in the room itself. There might be more, but that's what I found so far.

**Comment by ReynTheLord** [comment]
![image](https://user-images.githubusercontent.com/37010048/105735045-7ef3c900-5ef0-11eb-9610-1b28c5d540a1.png)
 Relevant image

**Comment by ghost** [comment]
This has long been fixed and can be closed @Funce (i think i @ you?)

**Comment by Funce** [comment]
Yep! Thats the one! All good!
